packages/itwinui-css/backstop/tests/dialog.html

```diff
@@ -75,11 +75,9 @@ <h1>Dialog</h1>
       Toggle draggable dialog
     </button>
 
-    <div
-      class="iui-dialog-backdrop iui-dialog-default"
-      id="default-dialog"
-    >
-      <div class="iui-dialog">
+    <div id="default-dialog">
```

Comment:
Shouldn't we have some wrapper? We could even use it for later to center dialog using flex.

Reply:
Related (conversation)[https://github.com/iTwin/iTwinUI/pull/670#issuecomment-1145957843].

We have `wrapper` for contained dialog. It might be hard to think other name.